src/core/js/views/drawerView.js

```diff
@@ -27,7 +27,7 @@ define(function(require) {
         setupEventListeners: function() {
             this.listenTo(Adapt, 'navigation:toggleDrawer', this.toggleDrawer);
             this.listenTo(Adapt, 'drawer:triggerCustomView', this.openCustomView);
-            this.listenToOnce(Adapt, 'adapt:initialize', this.checkIfDrawerIsAvailable);
+            this.listenTo(Adapt, 'app:dataReady', this.checkIfDrawerIsAvailable);
```

Comment:
now that `checkIfDrawerIsAvailable` has been added to `postRender` it actually works fine for me without this line